test/e2e/storage/csi_volumes.go

```diff
@@ -237,4 +240,61 @@ var _ = utils.SIGDescribe("CSI Volumes [Flaky]", func() {
 			testDynamicProvisioning(t, cs, claim, class)
 		})
 	})
+
+	Describe("Sanity CSI plugin test using GCE-PD CSI driver", func() {
+		var (
+			controllerClusterRoles []string = []string{
+				csiExternalAttacherClusterRoleName,
+				csiExternalProvisionerClusterRoleName,
+			}
+			nodeClusterRoles []string = []string{
+				csiDriverRegistrarClusterRoleName,
+			}
+			controllerServiceAccount *v1.ServiceAccount
+			nodeServiceAccount       *v1.ServiceAccount
+		)
+
+		BeforeEach(func() {
+			framework.SkipUnlessProviderIs("gce", "gke")
+			// Currently you will need to manually add the required GCP Credentials as a secret "cloud-sa"
+			// kubectl create generic cloud-sa --from-file=PATH/TO/cloud-sa.json --namespace={{config.Namespace}}
+			// TODO(GITHUBISSUE): Inject the necessary credentials automatically to the driver containers in e2e test
+			framework.SkipUnlessSecretExistsAfterWait(cs, "cloud-sa", config.Namespace, 3*time.Minute)
```

Comment:
I'm assuming this can't be run in CI for now? Maybe just put a `[Feature:]` tag to skip it in CI.

Reply:
done